src/native/fs/fs_napi.cpp

```diff
@@ -102,7 +103,12 @@ struct FSWorker : public Napi::AsyncWorker
         DBG1("FS::FSWorker::Execute: " << _desc << DVAL(_uid) << DVAL(_gid) << DVAL(_backend));
         ThreadScope tx;
         tx.set_user(_uid, _gid);
+        std::time_t timeBefore = std::time(nullptr);
```

Comment:
time_t holds the number of seconds since the epoch.
I'm not sure if that is the granularity that we want to use, I would see if we could have something at least in the millisecond range.